python/sort_number.py

```diff
@@ -0,0 +1,23 @@
+import sys
+#This function will sort a given sequence of numbers
```

Comment:
It is recommended in the Python guidelines that between the imports and every function, there is two empty lines.